guides/installation/overview.md

```diff
@@ -0,0 +1,182 @@
+# Installation Overview
```

Comment:
Why does the PR add this article? I think it was removed by Christian when restructuring the installation section.

Reply:
It got added when merging conflicts.

Reply:
Infact this file was not removed but added/modified

Reply:
Yes you were right. I have now removed this file.